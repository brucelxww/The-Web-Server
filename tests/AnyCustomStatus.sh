cd ..
python3 webserv.py config.cfg &
PID=$!
cd -
sleep 2
curl -i localhost:8070/cgibin/teststatus.py | diff - StatusAnyNumber.out 
kill $PID