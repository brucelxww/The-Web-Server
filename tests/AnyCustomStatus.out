HTTP/1.1 666 Custom Status
Content-Type: text/html

Status-Code: 666 CSCSCS
<h1>Hello From Python!</h1>
